src/textures/Texture.js

```diff
@@ -337,6 +337,15 @@ class Texture extends EventDispatcher {
 		 */
 		this.isRenderTargetTexture = false;
 
+		/**
+		 * Indicates if a texture should be handled like a texture array.
+		 *
+		 * @type {boolean}
+		 * @readonly
+		 * @default false
+		 */
+		this.isTextureArray = false;
```

Comment:
@Mugen87 I don't really like all the `isTextureArray` checks everywhere. Do you think there's a cleaner way?

Reply:
I think we need to organize our texture classes in a different manner to avoid such issues. Maybe we have to introduce `ArrayTexture` as a base class for `DataArrayTexture`. `ArrayTexture` could then be used as an alternative render target texture of `XRRenderTarget`. 

I'm not familiar with multiview in detail but that does that means when using multiview a different render target setup is required? Related question: https://github.com/mrdoob/three.js/pull/30920/files#r2041078892 but does th

Reply:
The point of multiview is cutting down the number of gl calls by half.
Because the majority of the draw call happen when we render to the non-xr drawtarget, we need to teach it to use multiview.